overcast/django/apps/buildsvc/pkgbuild/__init__.py

```diff
@@ -13,12 +14,14 @@
 from ..models import BuildRecord
 
 class PackageBuilder(object):
-    def __init__(self, basedir, package_source, build_record):
+    def __init__(self, basedir, package_source, build_record,
+                 docker_build=True):
```

Comment:
also i just kept an option to use sbuild if there is any such usecase or situation